packages/kbn-io-ts-utils/src/route_validation/index.ts

```diff
@@ -0,0 +1,53 @@
+/*
```

Comment:
Copied from 
https://github.com/elastic/kibana/blob/main/x-pack/plugins/infra/common/runtime_types.ts#L58

https://github.com/elastic/kibana/blob/main/x-pack/plugins/monitoring/server/lib/create_route_validation_function.ts#L0-L1

The idea is to this function it here and remove those duplications.